src/components/WebexInMeeting/__snapshots__/WebexInMeeting.stories.storyshot

```diff
@@ -22,12 +22,6 @@ Array [
       <div
         className="wxc wxc-remote-media wxc-in-meeting__remote-media-in-meeting wxc-remote-media--remote-video-n-share"
       >
-        <video
```

Comment:
Why has this been removed from the snapshots? Comment applies to other places where there are deletions as well.

Reply:
This happened because we wrapped the remote video with `Draggable` component which is mocked in the test file.

Reply:
Can we look for the Draggable component at least in the snapshot? It should be rendered right even if it's a mock object?

Reply:
Updated the mock, now it includes the Draggable and it's children nodes